core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -7125,7 +7366,7 @@ components:
           format: int32
           minimum: 0
           maximum: 10000
-          description: An integer between `0` and `10000`, giving the total count of transactions in the returned response
+          description: An integer between `0` and `10000`, giving the total count of proofs in the returned response
```

Comment:
AFAIS, this is a part of `StreamTransactionsResponse` and it rightfully mentioned `transactions` count.

Reply:
Ooops, good spot, thanks!